src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -41,6 +44,31 @@
   void close();
 
   /**
+   * @return the remote address for this socket
+   */
+  @CacheReturn
+  SocketAddress remoteAddress();
+
+  /**
+   * @return the local address for this socket
+   */
+  @CacheReturn
+  SocketAddress localAddress();
```

Comment:
Ditto as above.

Reply:
It is useful for logging purposes. I can create another PR if you wish for that functionality.

Reply:
@dachaac yes. It's better having a different PR for that. Thanks !

Reply:
@ppatierno Created new PR for that: https://github.com/vert-x3/vertx-mqtt/pull/56

Will remove it from this PR now.